app/onboarding/page.tsx

```diff
@@ -100,41 +115,94 @@ const PlotSelection: React.FC<PlotSelectionProps> = ({
     </div>
   );
 };
+const stateOptions: DropdownOption<string>[] = [
+  { label: 'Tennessee', value: 'TENNESSE' },
+  { label: 'Missouri', value: 'MISSOURI' },
+];
 
-// Main Onboarding Component
-const OnboardingFlow = () => {
-  const [step, setStep] = useState(1);
-  const [selectedState, setSelectedState] = useState<string>('');
-  const [selectedGardenType, setSelectedGardenType] = useState<string>('');
-  const [selectedPlot, setSelectedPlot] = useState<boolean>(false);
+const gardenTypeOptions: DropdownOption<string>[] = [
+  { label: 'Individual', value: 'Individual' },
+  { label: 'Community', value: 'Community' },
+  { label: 'School', value: 'School' },
+];
 
-  const handleNext = () => {
-    setStep(step + 1);
-  };
+const plot: DropdownOption<boolean>[] = [
+  { label: 'Yes, I own a plot', value: true },
+  { label: "No, I don't own a plot", value: false },
+];
 
-  const handleBack = () => {
-    setStep(step - 1);
-  };
+// Updated ReviewPage component to accept necessary props
 
+const ReviewPage = ({
+  selectedState,
+  setSelectedState,
+  selectedGardenType,
+  setSelectedGardenType,
+  selectedPlot,
+  setSelectedPlot,
+}: ReviewPageProps) => {
   const handleSubmit = async () => {
     const profile: Profile = {
       user_id: '2abd7296-374a-42d1-bb4f-b813da1615ae',
-      state: selectedState,
+      us_state: selectedState,
       user_type: selectedGardenType,
-      has_plot: selectedPlot,
+      // has_plot: selectedPlot,
     };
+
     try {
-      upsertProfile(profile);
+      await upsertProfile(profile);
+      console.log('Profile submitted successfully:', profile);
     } catch (error) {
       console.error('Error upserting profile:', error);
-      throw new Error('Error upserting profile');
-    } finally {
-      //TODO: Remove console log.
-      console.log('Submitted data: ', profile);
     }
-    // Handle form submission, e.g., send to a server or display a confirmation
   };
 
+  return (
+    <PageContainer>
+      <ReviewContainer>
+        <H3 style={{ textAlign: 'center' }}>Review Your Response</H3>
```

Comment:
nit: we'll need to create global styles eventually - i'm planning to add that commit to main soon whenever the design system is done, but in ur definition of H3, you can make it a `styled.h3` rather than a `styled.p`